### PR TITLE
gviz:0.1.0

### PR DESCRIPTION
**Pull request by SekoiaTree**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: This is a package to draw graphs in general. There's no package for this yet because graph layouting is very difficult, so plugins were needed. This uses the graphviz dot language, and a fork of layout-rs.

### CODE REVIEW

File: packages/preview/gviz/0.1.0/gviz.typ [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+///
+/// - code (string, bytes): Dot language code to be rendered.
+/// - width (auto, relative): The width of the image.
+/// - height (auto, relative): The height of the image.
+/// - alt (none, string): A text describing the image.
+/// - fit (string): How the image should adjust itself to a given area. See image.decode.
+/// -> content
+#let render-image(code, width: auto, height: auto, alt: none, fit: "cover") = {
+  image.decode(render(code), format: "svg", width: width, height: height, alt: alt, fit: fit)
+}
```
Comment (laurmaedje) [inline packages/preview/gviz/0.1.0/gviz.typ:25]: I would suggest forwarding all arguments. If you provide manual defaults like `fit: "cover"`, image set rules for that won't work anymore. It's also more future-proof.
```suggestion
#let render-image(code, ..args) = {
  image.decode(render(code), format: "svg", ..args)
}
```

File: packages/preview/gviz/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# GViz
+GViz is a typst plugin that can render graphviz graphs.
+
+It uses https://codeberg.org/Sekoia/layout as a backend, which means it can currently only render to SVG, and mostly supports basic features.
+
+Import it like any other plugin: `#import "@preview/gviz:0.1.0": *`.
+
+## Usage
+```typst
+#import "@preview/gviz:0.1.0: *"
```
Comment (laurmaedje) [inline packages/preview/gviz/0.1.0/README.md:10]:
```suggestion
#import "@preview/gviz:0.1.0": *
```

**Comment by SekoiaTree** [comment]
Fixed!

**Comment by istudyatuni** [comment]
Maybe add source code for plugin?

**Comment by SekoiaTree** [comment]
I already have, it's in typst.toml. https://codeberg.org/Sekoia/gviz-typst

**Comment by laurmaedje** [comment]
Thanks! First plugin 🎉

**Pull request merged**